documentation/book/getting-started.adoc

```diff
@@ -12,21 +12,24 @@ include::common/attributes.adoc[]
 [id='getting-started-{context}']
 = Getting started
 
-{ProductLongName} runs on {ProductPlatformName}.
-{ProductName} supports
-ifdef::Kubernetes[{KubernetesLongName} {KubernetesVersion} or]
+{ProductName} runs on
+ifdef::Kubernetes[{KubernetesLongName} {KubernetesVersion} and]
 {OpenShiftLongName} {OpenShiftVersion}.
-{ProductName} works on all kinds of clusters - from public and private clouds down to local deployments intended for development.
+It can be used on clusters of public and private clouds, local deployments intended for development, and everything in between.
+
 This guide expects that an {ProductPlatformName} cluster is available and the
-ifdef::Kubernetes[`kubectl` or]
+ifdef::Kubernetes[`kubectl` and]
 `oc` command line tools are installed and configured to connect to the running cluster.
 
 ifdef::InstallationAppendix[]
 When no existing {ProductPlatformName} cluster is available, `Minikube` or `Minishift` can be used to create a local
-cluster. More details can be found in <<installing_kubernetes_and_openshift_cluster>>
+cluster. More details can be found in xref:installing_kubernetes_and_openshift_cluster[Installing Kubernetes and OpenShift clusters].
 endif::InstallationAppendix[]
 
-In order to execute the commands in this guide, your {ProductPlatformName} user needs to have the rights to create and
+[NOTE]
+To run the commands in this guide, your
+ifdef::Kubernetes[{KubernetesLongName} and]
+{OpenShiftLongName} user needs to have the rights to create and
 manage Roles and Role Bindings (RBAC) resources.
```

Comment:
I thought "RBAC" stood for "role-based access control" ?

Reply:
And you are right :) I guess that the interpretation is : manage Role and Role Bindings resources which are part of the RBAC. But I can understand your confusion.

Reply:
Actually, we now require Cluster Roles and Cluster Role Bindings. It might be easier for future maintenance to say _... user needs to have the rights to manage role-based access control (RBAC)._